waku/v2/protocol/rln/pb/rln.pb.go

```diff
@@ -0,0 +1,202 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.31.0
+// 	protoc        v3.21.12
```

Comment:
This file seems to be still using older version of protoc, whereas other files are generated using protoc  v4.24.4.
Better to verify all files once for protoc version used to generate. Did not check each and every file, but randomly verified this one and found out version difference.